fastapi/routing.py

```diff
@@ -627,6 +628,16 @@ def include_router(
             responses = {}
         for route in router.routes:
             if isinstance(route, APIRoute):
+                if "GET" in route.methods and "HEAD" not in route.methods:
```

Comment:
Reverse the condition and add the continue statement to decrease the indentation level like the following.

```py
if "GET" not in route.methods or "HEAD" in route.methods:
    continue

#your code goes here
```